### DISCUSSION TITLE
Integrating TCR data?

### DISCUSSION THREAD

**Discussion by bl24** [satijalab/seurat#5363] [Q&A]
Hello all,

I'm working on a 10x TGEX dataset that has corresponding TCR sequencing. What I would like to do is use the TCR data to figure out the 10 most abundant clonotypes and then somehow cluster the TGEX cells based on those clonotypes.

Is this doable?

**Comment by samuel-marsh** [comment]
Hi,

Yes this type of analysis is doable but is not a feature of Seurat.  I would suggest checking out the [scRepertoire](https://github.com/ncborcherding/scRepertoire) or other TCR/BCR specific analysis packages (there are others out there too).

Best,
Sam

**Comment by samuel-marsh** [comment]
For clustering based on clonotype may want to check out CoNGA https://github.com/phbradley/conga
https://www.nature.com/articles/s41587-021-00989-2
